packages/hash/playwright/playwright.config.ts

```diff
@@ -8,7 +8,6 @@ const config: PlaywrightTestConfig = {
   timeout: 60000,
   projects: [
     { name: "chromium", use: { ...devices["Desktop Chrome"] } },
-    { name: "firefox", use: { ...devices["Desktop Firefox"] } },
```

Comment:
Can we add a `@todo` that explains:
- why we've done this
- links to the Asana task
- explains on what condition we will undo it

then we can undo at some point and make sure that the task is ticked off at the same time

I suggest we should do this as a matter of course to avoid having tasks and todos unlinked and become out of sync

Reply:
Done in https://github.com/hashintel/hash/pull/1317/commits/75237c5ed96e3c35f677e9d794b5be9330d73b0a